dev.rollup.config.js

```diff
@@ -26,7 +26,7 @@ try {
 const transformExampleSdkUrl = (buffer) => {
     return buffer.replace(
         /(\s*<script.*src=")([^"]+client-app-sdk[^"]+)(".*<\/script>\s*)/i,
-        '$1' + (BROWSER_FILENAME || '$2') + '$3\n'
+        '$1' + (`/${BROWSER_FILENAME}` || '$2') + '$3\n'
```

Comment:
Noticed this tiny bug while in here, need to prepend a `/` to the path otherwise the examples with a nested folder structure won't be able to find the sdk.

Reply:
Ah, nice catch!